pkg/api/validation/schema.go

```diff
@@ -88,6 +101,29 @@ func (s *SwaggerSchema) ValidateBytes(data []byte) error {
 	if kind == nil {
 		return fmt.Errorf("kind not set")
 	}
+	if kind.(string) == "List" {
```

Comment:
@smarterclayton runtime.IsList would go where exactly? In a new file in pkg/runtime?